core/src/main/java/org/akaza/openclinica/bean/managestudy/EventDefinitionCRFBean.java

```diff
@@ -579,38 +577,38 @@ public void setEventName(String eventName) {
         this.eventName = eventName;
     }
 
-	public CRFVersionBean getDefaultCRF() {
-		return defaultCRF;
-	}
+    public FormLayoutBean getDefaultCRF() {
+        return defaultCRF;
+    }
+
+    public void setDefaultCRF(FormLayoutBean defaultCRF) {
+        this.defaultCRF = defaultCRF;
+    }
 
-	public void setDefaultCRF(CRFVersionBean defaultCRF) {
-		this.defaultCRF = defaultCRF;
-	}
+    public boolean isParticipantForm() {
+        return participantForm == true;
```

Comment:
Not sure if I understand. Why do we need 
 participantForm == true;?

Can't you just return  participantForm?